docs/installation/configuration.rst

```diff
@@ -0,0 +1,63 @@
+.. _configuration:
+
+=======================
+Open Zaak configuration
+=======================
+
+Before you can work with Open Zaak after installation, a few settings need to be
+configured first.
+
+.. _configuration_sites:
+
+Setting Sites.Site
+==================
+
+In the admin, under **Configuration > Sites**, make sure to change the existing
+``Site`` to the domain under which Open Zaak will be deployed (see :ref:`manual_configuration` for more
+information).
+
+Configure API-access between components
+=======================================
+
+To allow communication between the different APIs in Open Zaak, e.g. between Zaken and Catalogi, etc.,
+follow these steps:
+
+- Create ``Client credentials`` for each component (**API Authorizations > Client credentials**)
```

Comment:
**Client credentials** was removed and is only accessible via **API Authorizations > Applicaties**